test/unit/services/test_image.py

```diff
@@ -160,14 +162,24 @@ class TestCreate(CreateDSL):
     def test_create(self):
         """Test creating an image."""
 
-        self.mock_create(IMAGE_POST_METADATA_DATA_ALL_VALUES)
+        self.mock_create(IMAGE_POST_METADATA_DATA_ALL_VALUES, "test.png")
         self.call_create()
         self.check_create_success()
 
+    def test_create_with_invalid_file(self):
```

Comment:
Could update this test name so its more explicit about what is invalid (e.g. example above).